drivers/hmis/app/models/hmis/hud/client.rb

```diff
@@ -126,4 +139,9 @@ def self.sort_by_option(option)
   def age(date = Date.current)
     GrdaWarehouse::Hud::Client.age(date: date, dob: self.DOB)
   end
+
+  def image
+    # TODO: Remove this when stub is no longer needed
+    fake_client_image_data
+  end
```

Comment:
@eanders will re-write this.  Current thinking is that we'll pull `GrdaWarehouse::ClientFile` that belongs to the source client (currently all belong to the destination client) with a particular tag.  
Decisions:
1. It's ok that the file is attached to the source client
2. This mechanism shouldn't take any other warehouse files into account, we'll handle that when we build the warehouse endpoint